axiom/tests/test_txn.c

```diff
@@ -8493,6 +8510,10 @@ static void test_txn_log_configuration(void) {
   txn->options.log_forwarding_enabled = true;
   txn->options.log_events_max_samples_stored = 1;
   tlib_pass_if_false(__func__, nr_txn_log_forwarding_enabled(txn), "global=1, high_security=1, forwarding=1, samples=1 -> off");
+
+  txn->options.log_forwarding_enabled = false;
+  txn->options.log_decorating_enabled = true;
+  tlib_pass_if_true(__func__, nr_txn_log_decorating_enabled(txn), "global=1, high_security=1, decorating=1 -> on");
```

Comment:
Do we have any tests verifying both log forwarding and log decoration being enabled?

Reply:
Added unit test in commit [b9ec16e](https://github.com/newrelic/newrelic-php-agent/pull/733/commits/b9ec16ecac3f775fd0bf98733cdd68fcca02d18a).

Reply:
Added integration tests in commit [fdb93dc](https://github.com/newrelic/newrelic-php-agent/pull/733/commits/fdb93dc8f1a9ca4d4def9272330dba6a81b28c5f).